### PR TITLE
ResNet164 || CIFAR10 || Ryan

### PR DESCRIPTION
**Pull request by lxylzxc**
Ryan submissions for CIFAR10 train and inference with resnet164 without bottleneck on x86.

### CODE REVIEW

File: CIFAR10/train/ryan_resnet164_nb_1p100-dawn_tensorflow.tsv [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+93.08928	1.9617921262979507	92.60000000000001
+94.09024	1.9828753029637867	92.81
+95.0912	2.0039472704463535	92.84
+96.09216	2.0250190403064092	92.99
+97.09312	2.0461121825377147	92.72
+98.09408	2.067193559077051	92.97
+99.09504	2.0882425423463187	92.69
+100.096	2.109299462040265	92.95
+101.09696	2.1303626803557076	92.75
+102.09792	2.1514113063282436	93.05
```
Comment (codyaustun) [inline CIFAR10/train/ryan_resnet164_nb_1p100-dawn_tensorflow.tsv]: @lxylzxc we are only reviewing results that are in the [top-5 results for at least one leaderboard](https://dawn.cs.stanford.edu/2018/07/25/dawnbench_rolling/). It doesn't look like this is fast enough to be considered for training in terms of time or cost. Can you remove the CIFAR10/train files from this pull request?

**Review by codyaustun — CHANGES_REQUESTED** [review]
The inference results look okay, but please remove the training results because they don't qualify for review under rolling submissions.

File: CIFAR10/train/ryan_resnet164_nb_1p100-dawn_tensorflow.tsv (hunk 1 above)
Comment (lxylzxc) [reply CIFAR10/train/ryan_resnet164_nb_1p100-dawn_tensorflow.tsv]: thanks for  your reply, I will remove the CIFAR10/train files.

**Review by codyaustun — APPROVED** [review]

**Comment by codyaustun** [comment]
Looks good to me. Thanks!

**Pull request merged**